aiida/cmdline/commands/cmd_computer.py

```diff
@@ -501,43 +491,70 @@ def computer_test(user, print_traceback, computer):
     num_failures = 0
     num_tests = 0
 
+    tests = {
+        _computer_test_no_unexpected_output: 'Checking for spurious output',
+        _computer_test_get_jobs: 'Getting number of jobs from scheduler',
+        _computer_get_remote_username: 'Determining remote user name',
+        _computer_create_temp_file: 'Creating and deleting temporary file'
+    }
+
     try:
-        echo.echo('> Testing connection...')
+        echo.echo('* Opening connection... ', nl=False)
+
         with transport:
-            scheduler.set_transport(transport)
             num_tests += 1
-            for test in [_computer_test_no_unexpected_output, _computer_test_get_jobs, _computer_create_temp_file]:
+
+            click.secho('[OK]', fg=echo.COLORS['success'], bold=echo.BOLD)
```

Comment:
in my 2nd PR I introduced an `echo.echo_highlight` command that saves one from specifying the `bold` parameter + one can pass directly `'success'`.
Can save some text...

Reply:
I have added this to your first commit and then used it in mine.